GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -188,6 +195,76 @@ gdt_cct_IosClientInfo GDTCCTConstructiOSClientInfo() {
   return iOSClientInfo;
 }
 
+gdt_cct_NetworkConnectionInfo GDTCCTConstructNetWorkConnectionInfo() {
```

Comment:
Something to consider: if the network connection info struct ever has a dynamic length member added to it, some of this code will break. Consider returning NSData from this method to help address this problem by making the problem manifest here, as opposed to a consumer of the API.